versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/objtypes/Hashes.java

```diff
@@ -80,9 +80,8 @@ static ObjId refHash(String name, ObjId initialPointer, long createdAtMicros) {
     return hashAsObjId(hasher);
   }
 
-  static ObjId tagHash(
-      ObjId commitId, String message, CommitHeaders headers, ByteString signature) {
-    Hasher hasher = newHasher().putString(TAG.name(), UTF_8).putBytes(commitId.asByteArray());
+  static ObjId tagHash(String message, CommitHeaders headers, ByteString signature) {
+    Hasher hasher = newHasher().putString(TAG.name(), UTF_8);
```

Comment:
Just curious: is there any consequence to the fact that the hash of tag objects has changed? E.g. if we recompute the hash of an existing tag that was hashed with the previous code, we won't find the same hash.

Reply:
The tag-objects have never been used. So this breaking change is fine.